packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -4,6 +4,90 @@ title: Okta Identity Engine API Products release notes 2022
+
+#### Access Denied error message customization is GA in Preview
+
+Admins can now customize the error message that users receive when their access is denied. This allows admins to provide remediation steps and/or point users to documentation that helps resolve their access issues.<!--OKTA-512724-->
+
+#### Dynamic IdP routing is GA in Preview
+
+Org admins can now consolidate multiple IdP routing rules into a single dynamic routing rule. Dynamic routing rules use expression language to match users to any IdP, based on attributes of their login object. This reduces the volume and complexity of routing rules and the manual effort of managing them. See [Policy Action with Dynamic IdP routing](/docs/reference/api/policy/#policy-action-with-dynamic-idp-routing).<!--OKTA-520971-->
+
+#### The update logo for application operation is updated
+
+The [update logo for application](/docs/reference/api/apps/#application-logo-operations) API operation has been updated to support logo images in SVG format. In addition, the updated logo guidelines require a square image dimension of 200px by 200px to avoid distortion.<!--OKTA-511103-->
+
+#### Clone authentication policies are GA in Preview
+
+Creating an authentication policy from scratch is a manual, error-prone task because you need to visually copy existing rules into the new policy. Okta now offers the ability to clone a policy. You can use either the Admin Console or the new Clone a Policy operation on the Policy API. See [Clone a Policy](/docs/reference/api/policy/#clone-a-policy).<!--OKTA-515109-->
+
+#### Improvements to the self-service password reset experience are GA in Preview
+
+Previously, the self-service password reset (SSPR) flow created unnecessary friction in the user experience. The newly enhanced SSPR feature introduces a seamless magic link experience for password reset emails. Users no longer need to provide consent when using the same browser. After a successful password reset where the password meets the application’s assurance policy, the user is signed directly to the app. See [Configure the Email authenticator](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-email).<!--OKTA-499514-->
+
+#### Password restriction for shared SWA app accounts is GA in Production
+
+For SWA apps with an account sign-in option set to Users share a single username and password set by administrator, only super admins or app admins with permissions for that app can view the password.<!--OKTA-513421-->
```

Comment:
This one was hard to digest. I'm suggesting putting the setting in bold so that it's easier to read.
```suggestion
For SWA apps with an account sign-in option set to **Users share a single username and password set by administrator**, only super admins or app admins with permissions for that app can view the password.<!--OKTA-513421-->
```